lib/src/source/git.dart

```diff
@@ -808,13 +821,13 @@ class GitDescription extends Description {
   }
 }
 
-class GitResolvedDescription extends ResolvedDescription {
+class ResolvedGitDescription extends ResolvedDescription {
```

Comment:
This rename is completely orthogonal to the rest of the PR, right?

Should we consider doing it in a separate PR upfront, so there is less to review here?

(the rename seems like something that would be easy to review and land)